r/src/recordbatch.cpp

```diff
@@ -131,7 +131,7 @@ std::shared_ptr<arrow::RecordBatch> ipc___ReadRecordBatch__InputStream__Schema(
 arrow::Status check_consistent_array_size(
     const std::vector<std::shared_ptr<arrow::Array>>& arrays, int64_t* num_rows) {
   *num_rows = arrays[0]->length();
-  for (int64_t i = 1; i < arrays.size(); i++) {
+  for (size_t i = 1; i < arrays.size(); i++) {
```

Comment:
Idiomatic C++11 style would be something like this:
```cpp
for (const auto& array : arrays) {
  if (array->length() != *num_rows) {
    // etc.
```

Which also avoids having to think about index type.

Reply:
Thanks.